wrapper/src/rabbitmq/rabbitmq.cpp

```diff
@@ -100,7 +100,7 @@ RabbitMQ::handleIncomingMessages()
                     Match match = std::get<Match>(data);
                     std::string side =
                         match.side == messages::SIDE::BUY ? "BUY" : "SELL";
-                    nutc::pywrapper::get_trade_update_function()(
+                    nutc::pywrapper::get_trade_and_account_update_function()(
```

Comment:
Why do you need to get the function every time? Why not cache it in a variable?

Reply:
Don't know, I didn't write this lol

Reply:
We can't cache strategy because it's stateful, we *could* cache the attribute "on_trade_and_account_update" and other callbacks but I'm worried there's some nuance to caching it that will affect the program

Since we don't have unit tests to verify (lol) I'm gonna say keep it as is